test/module/Entra/Remove-EntraAdministrativeUnitMember.Tests.ps1

```diff
@@ -0,0 +1,41 @@
+BeforeAll {  
```

Comment:
Add license information at the top of this file

Reply:
done